oak_functions/loader/src/grpc.rs

```diff
@@ -82,21 +82,34 @@ pub async fn create_and_start_grpc_server<F: Future<Output = ()>>(
 
     let additional_info = config_info.encode_to_vec();
 
-    tonic::transport::Server::builder()
-        .add_service(
-            grpc_unary_attestation::proto::unary_session_server::UnarySessionServer::new(
-                grpc_unary_attestation::server::AttestationServer::create(
-                    tee_certificate,
-                    request_handler,
-                    additional_info,
-                    ErrorLogger { logger },
-                )
-                .context("Couldn't create remote attestation server")?,
-            ),
-        )
-        .serve_with_shutdown(*address, terminate)
-        .await
-        .context("Couldn't start server")?;
+    let grpc_unary_attestation_service =
+        grpc_unary_attestation::proto::unary_session_server::UnarySessionServer::new(
+            grpc_unary_attestation::server::AttestationServer::create(
+                tee_certificate,
+                request_handler,
+                additional_info,
+                ErrorLogger { logger },
+            )
+            .context("Couldn't create remote attestation server")?,
+        );
+
+    // Create a server and add the relevant service defintions. Server creation
+    // and start is handled entirely witin each respective conditional arm, as
+    // the added service alters the type signature of the created server.
+    if cfg!(feature = "oak-web") {
+        tonic::transport::Server::builder()
+            .add_service(grpc_unary_attestation_service.clone())
```

Comment:
Is it required to add the service twice? I would have expected `enable` to add support for grpc-web requests without breaking normal gRPC.

Reply:
You're right, the documentation on it isn't super clear, but it does pass through non grpc-web requests to the underlying service. 

https://github.com/hyperium/tonic/blob/350ccf95e4ee71f85a2a1af7dc3e1df6cca1a2e3/tonic-web/src/service.rs#L158

I probably would've preferred if the enable method didn't do that (or exposed an option to not do that), but it is what it is. Updated.